app/public/stylesheets/sass/_components/_view-container.scss

```diff
@@ -34,4 +34,9 @@
     &:last-child {
       margin-right: 0;
     }
-}
\ No newline at end of file
+}
+
+.streamView_inner {
+  margin-top: 50px;
+  margin-bottom: -50px;
+}
```

Comment:
Haha thanks for the reviewing.. yeah seemed weird `margin-top: 50` and `margin-bottom: -50`